ras_backstage/controllers/collection_exercise_controller.py

```diff
@@ -41,3 +41,26 @@ def get_collection_exercises_by_survey(survey_id):
 
     logger.debug('Successfully retrieved collection exercises', survey_id=survey_id)
     return json.loads(response.text)
+
+
+def get_linked_sample_summary_id(collection_exercise_id):
+    logger.debug('Retrieving sample linked to collection exercise', collection_exercise_id=collection_exercise_id)
+    url = f'{app.config["RM_COLLECTION_EXERCISE_SERVICE"]}collectionexercises/link/{collection_exercise_id}'
+    response = request_handler('GET', url, auth=app.config['BASIC_AUTH'])
+
+    if response.status_code == 204:
+        logger.error('No samples linked to collection exercise', collection_exercise_id=collection_exercise_id)
```

Comment:
This should just return None instead of raising an api error because if there is no linked samples we still want the collection exercise details page to load just with an upload sample option.